tests/src/model/pages/images-page.ts

```diff
@@ -78,12 +78,14 @@ export class ImagesPage extends MainPage {
   }
 
   async waitForImageExists(name: string): Promise<boolean> {
-    await waitUntil(async () => await this.imageExists(name), 3000, 900);
+    const imageExists = () => this.imageExists(name);
+    await waitUntil(imageExists, 3000, 900);
     return true;
   }
 
   async waitForImageDelete(name: string): Promise<boolean> {
-    await waitWhile(async () => await this.imageExists(name), 3000, 900);
+    const imageExists = () => this.imageExists(name);
+    await waitWhile(imageExists, 3000, 900);
```

Comment:
this looks unrelated to tests, just some personal preferences so it should be extracted to a separate PR